docs/operator-docs/lifecycle-policies.md

```diff
@@ -0,0 +1,152 @@
+---
+id: lifecycle-policies
+title: Lifecycle Policies
+hide_title: true
+---
+
+# Setting Lifecycle Policies
+
+This document will outline, in brief, how to set data lifecycle policies so that you are more easily able to control data storage costs while keeping your data accessible.
+
+## Requirements
+
+* The AWS CLI installed and configured (if you wish to run the CLI example).  See [AWS's guide to setting up the AWS CLI](https://docs.aws.amazon.com/AmazonS3/latest/dev/setup-aws-cli.html) for more on this.   Please ensure the AWS CLI is in your shell path.
+* You will need a S3 bucket on AWS. ***You are strongly encouraged to use a bucket without voluminous amounts of data in it for experimenting/learning***.
+* An AWS user with the appropriate roles to access the target bucket as well as modify bucket policies.
+
+## Examples
+### Walkthrough on setting time-based S3IA bucket policy
+This example will give step-by-step instructions on updating a bucket's lifecycle policy to move all objects in the bucket from the default storage to S3IA after a period of 90 days.   Below are instructions for walking through configuration via the command line and the management console.
```

Comment:
We should probably say something here like 'S3 Infrequent Access (S3IA)' instead of jump into the abbreviation.

Reply:
👍  I've added that to both references